bitnami/milvus/Chart.yaml

```diff
@@ -48,4 +48,4 @@ maintainers:
 name: milvus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/milvus
-version: 4.2.0
+version: 4.2.1
```

Comment:
As you are changing parameters, you should be pushing the major version. In this case it should be `5.0.0`.

Reply:
noted